CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 This is a history of changes to clara-rules.
 
+# 0.20.0 SNAPSHOT
+* Add flag to omit compilation context after Session compilation for use in serialization and deserialization. [issue 422](https://github.com/cerner/clara-rules/issues/422)
```

Comment:
I think this reads in a misleading way. The "omit compilation context" would be something you would not want to set to true for serde, since it'd be broken.
Maybe:

> Add a flag to omit compilation context (used by the durability layer) after Session compilation to save space when not needed.

Reply:
You could conceivably like to have the `compile-ctx` for other types of debugging? I don't think the flag is directly coupled to serialization is my point. It's just one example (the main one) of where it can be used.